app/policies/casa_case_policy.rb

```diff
@@ -31,38 +31,37 @@ def sibling_cases
   end
 
   def update_contact_types?
-    is_supervisor? || is_admin?
+    admin_or_supervisor_same_org?
   end
 
   def update_birth_month_year_youth?
-    is_admin?
+    is_admin_same_org?
   end
 
   def update_date_in_care_youth?
-    is_supervisor? || is_admin?
+    admin_or_supervisor_same_org?
   end
 
   def update_emancipation_option?
-    is_in_same_org? && (
-    admin_or_supervisor? || is_volunteer_actively_assigned_to_case?
-  )
+    # This permission is used in the Emancipations controller
+    admin_or_supervisor_same_org? || is_volunteer_actively_assigned_to_case?
   end
 
   def assign_volunteers?
-    is_in_same_org? && admin_or_supervisor?
+    admin_or_supervisor_same_org?
   end
 
   def can_see_filters?
-    is_supervisor? || is_admin?
+    admin_or_supervisor_same_org?
   end
 
-  alias_method :update_case_number?, :is_admin?
-  alias_method :update_case_status?, :is_admin?
-  alias_method :update_court_date?, :admin_or_supervisor_or_volunteer?
-  alias_method :update_hearing_type?, :admin_or_supervisor_or_volunteer?
-  alias_method :update_judge?, :admin_or_supervisor_or_volunteer?
-  alias_method :update_court_report_due_date?, :admin_or_supervisor_or_volunteer?
-  alias_method :update_court_orders?, :admin_or_supervisor_or_volunteer?
+  alias_method :update_case_number?, :is_admin_same_org?
+  alias_method :update_case_status?, :is_admin_same_org?
+  alias_method :update_court_date?, :admin_or_supervisor_or_volunteer_same_org?
+  alias_method :update_hearing_type?, :admin_or_supervisor_or_volunteer_same_org?
+  alias_method :update_judge?, :admin_or_supervisor_or_volunteer_same_org?
+  alias_method :update_court_report_due_date?, :admin_or_supervisor_or_volunteer_same_org?
+  alias_method :update_court_orders?, :admin_or_supervisor_or_volunteer_same_org?
```

Comment:
I wonder if using `alias_method` here is the right approach. If it is, we should probably do it for most of the calls that simply return the value of another method call. If not, we shouldn't use it at all.

Reply:
Hi Dave, Just a quick clarification, what do you mean by "the calls that simply return the value of another method call"? Not to return values of methods that in turn return values from other methods?